dev-libs/hyprland-protocols/hyprland-protocols-0.2.ebuild

```diff
@@ -1,17 +1,23 @@
-# Copyright 2023 Gentoo Authors
+# Copyright 2023-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit meson
 
-COMMIT=4d29e48433270a2af06b8bc711ca1fe5109746cd
 DESCRIPTION="Wayland protocol extensions for Hyprland"
 HOMEPAGE="https://github.com/hyprwm/hyprland-protocols"
-SRC_URI="https://github.com/hyprwm/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
-S="${WORKDIR}/${PN}-${COMMIT}"
 
-KEYWORDS="amd64 ~riscv"
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/hyprwm/${PN}.git"
+	inherit git-r3
+else
+	COMMIT=4d29e48433270a2af06b8bc711ca1fe5109746cd
+	SRC_URI="https://github.com/hyprwm/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${COMMIT}"
+	KEYWORDS="amd64 ~riscv"
+fi
+
 LICENSE="BSD"
 SLOT="0"
 
```

Comment:
Empty RDEPEND and DEPEND below can be removed (it was my mistake)

Reply:
Ok I will do the changes tomorrow

Reply:
Done